packages/shared/dao/sagas.ts

```diff
@@ -198,7 +198,8 @@ function* initializeCatalystCandidates() {
 
 export async function checkValidRealm(baseUrl: string): Promise<PingResult | null> {
   const pingResult = await ping(baseUrl + '/about')
-  if (pingResult.status === ServerConnectionStatus.OK) {
+  const acceptingUsers = pingResult.result && (pingResult.result.acceptingUsers ?? true)
```

Comment:
Is this condition checked to return false when acceptingUsers == false?

Reply:
not sure to understand the question, sorry

Reply:
I think the doubt comes from:
```ts
(pingResult.result.acceptingUsers ?? true)
```

but

```ts
(false ?? true) = false
```

so it should be ok